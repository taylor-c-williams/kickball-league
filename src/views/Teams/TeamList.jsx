import React from 'react';

export default function TeamList() {
	return <div></div>;
}
