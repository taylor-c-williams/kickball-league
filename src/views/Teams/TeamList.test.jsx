import { render, screen } from '@testing-library/react';
import TeamList from './TeamList';

test('renders TeamList elements', () => {
	const { container } = render(<TeamList />);

	const textElement = screen.getByText('teams', { exact: false });
	expect(textElement).toBeInTheDocument();
	expect(container).toMatchSnapshot();
});
