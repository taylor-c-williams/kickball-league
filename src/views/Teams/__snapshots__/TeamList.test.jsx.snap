// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TeamList elements 1`] = `
<div>
  <div>
    Teams:
    <ul />
  </div>
</div>
`;
